driverconfig/config.yaml

```diff
@@ -108,14 +109,15 @@ drivers:
         external-health-monitor: gcr.io/k8s-staging-sig-storage/csi-external-health-monitor-controller:v0.4.0
   - name: isilon
     configVersions:
-      - configVersion: v2.4.0
+      - configVersion: v2.5.0
         useDefaults: true
         supportedVersions:
           - version: v121
           - version: v122
```

Comment:
We support n-2 k8s versions . Support for Other needs to be removed.

Reply:
Removed the 1.22 k8's support